docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+# Grid layout(Tizen 6.0 or late)
+
+> **Note**
+>
+> If you are using GridLayout on Tizen 5.5, Please refer to [Grid Layout(Tizen 5.5)](#gridlayout-tizen5.5) instead.
+
+`GridLayout` is a grid box for the two dimentional layout. It constraints the x and y position, width and height of the child actors.
+
+Positioning children in a grid form, the size of each column and row is set to the largest size of cells belonging to their axis.
```

Comment:
This sentence doesn't look correctly phrased. What are we trying to say here?

Reply:
It positions the children in a grid form, where the size of each column and row is set to the largest size of the cells belonging to their axis.